pages/tasks.js

```diff
@@ -32,13 +32,28 @@ const Tasks = ({ appContext }) => {
       api
         .getCombined(
           '/invitations',
-          { ...commonParams, replyto: true, details: 'replytoNote,repliedNotes', type: 'notes'},
-          { ...commonParams, replyto: true, details: 'replytoNote,repliedNotes,repliedEdits', type: 'note' },
+          {
+            ...commonParams,
+            replyto: true,
+            details: 'replytoNote,repliedNotes',
+            type: 'notes',
+          },
+          {
+            ...commonParams,
+            replyto: true,
+            details: 'replytoNote,repliedNotes,repliedEdits',
+            type: 'note',
+          },
           commonOptions
         )
         .then(addPropertyToInvitations('noteInvitation')),
       api
-        .getCombined('/invitations', { ...commonParams, type: 'tags' }, { ...commonParams, type: 'tag' }, commonOptions)
+        .getCombined(
+          '/invitations',
+          { ...commonParams, type: 'tags' },
+          { ...commonParams, type: 'tag' },
+          commonOptions
```

Comment:
Not sure why you are querying tag invitations, but you only need to pass one:
```suggestion
          '/invitations',
          { ...commonParams, type: 'tag' },
          commonOptions
```

Reply:
Correct, but you only need to pass singular. The API 2 supports plural to avoid breaking the UI, but it should be singular

Reply:
aaah I understand now. Yes, it's correct. No need to change anything.